EIPS/eip-000.md

```diff
@@ -171,6 +171,27 @@ The most expensive transaction contributes a different value because it determin
 A single transaction accessing an address or a slot that is not shared by other transactions does not trigger a refund,
 and therefore has a zero marginal contribution.
 
+### Efficiently storing the access lists in the block history
+
+The contents of the `accessList` parameter are part of the Ethereum history and the potential cost of keeping this
+data in the blockchain must be accounted for when implementing this change.
+There is currently no additional charge applied to the `accessList` parameter, due to the cost of including
+an address or a storage slot in the `accessList` being a constant value that is significantly higher than the
+potential cost of storing the `accessList` at the cost of  a
+dynamically sized `calldata` field.
+
+With the block-level warming there is a change that makes it possible for a transaction sender to construct
+transactions with a large `accessList` that cost very little to be included, and this can be used to bloat the
+blockchain size.
+
+In order to minimize the cost of permanently storing access lists, we suggest the execution clients to maintain
```

Comment:
We need more than suggestion, but actual transaction type that define this compressed structure, and define how each tx can reference it.
A slight modification to @yoavw compression is to compress separately addresses and slots.
If an access list address is 2 bytes,  it is a reference into the global table, instead of locally in this tx. If address is explicit (20 bytes) then it's slots are not compressed.

Reply:
> We need more than suggestion
Replaced "suggestion" with a required protocol change.

> but actual transaction type that define this compressed structure, and define how each tx can reference it.

Individual transactions, of any type, cannot "compress" their own access lists. Only on a block level we can apply the compression.

> A slight modification to @yoavw compression is to compress separately addresses and slots. If an access list address is 2 bytes, it is a reference into the global table, instead of locally in this tx. If address is explicit (20 bytes) then it's slots are not compressed.

I don't see a reason to mix-and-match compressed and uncompressed entities in the system. If there is a good use-case, please advise.